integration_test/tests/playbackTest.ts

```diff
@@ -0,0 +1,61 @@
+import { TestScope } from 'cavy';
+import {
+  callPlayerAndExpectEvent,
+  callPlayerAndExpectEvents,
+  EventSequence,
+  EventType,
+  expectEvents,
+  loadSourceConfig,
+  playFor,
+  RepeatedEvent,
+  startPlayerTest,
+} from '../playertesting';
+import { Sources } from './helper/Sources';
+
+export default (spec: TestScope) => {
+  spec.describe('calling player API when a source is loaded', () => {
+    spec.it(
+      'emits a Play and Playing events after calling play API',
+      async () => {
+        await startPlayerTest({}, async () => {
+          await loadSourceConfig(Sources.artOfMotionHls);
+          await callPlayerAndExpectEvents((player) => {
+            player.play();
+          }, EventSequence(EventType.Play, EventType.Playing));
+        });
+      }
+    );
+    spec.it('emits a Paused event after calling pause API', async () => {
+      await startPlayerTest({}, async () => {
+        await loadSourceConfig(Sources.artOfMotionHls);
+        await playFor(1);
+        await callPlayerAndExpectEvent((player) => {
+          player.pause();
+        }, EventType.Paused);
+      });
+    });
+  });
```

Comment:
This should be two separate blocks I think.
```suggestion
  spec.describe('calling play when a source is loaded', () => {
    spec.it(
      'emits a Play and Playing events',
      async () => {
        await startPlayerTest({}, async () => {
          await loadSourceConfig(Sources.artOfMotionHls);
          await callPlayerAndExpectEvents((player) => {
            player.play();
          }, EventSequence(EventType.Play, EventType.Playing));
        });
      }
    );
  });
  spec.describe('calling pause when a source is loaded', () => {
    spec.it('emits a Paused event', async () => {
      await startPlayerTest({}, async () => {
        await loadSourceConfig(Sources.artOfMotionHls);
        await playFor(1);
        await callPlayerAndExpectEvent((player) => {
          player.pause();
        }, EventType.Paused);
      });
    });
  });
```

Reply:
Applied feedback in 118b30c